### ISSUE TITLE
Error: column Contacts._bearing does not exist

### ISSUE THREAD

**Issue by BarisSari** [debrief/pepys-import#240]
Hi all. I'm in the develop branch and when I run the Import CLI, it throws the following error:

```bash
Traceback (most recent call last):
  File "/Users/baris/pepys-import/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 1245, in _execute_context
    self.dialect.do_execute(
  File "/Users/baris/pepys-import/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 588, in do_execute
    cursor.execute(statement, parameters)
psycopg2.errors.UndefinedColumn: column Contacts._bearing does not exist
LINE 2: ..., pepys."Contacts".time AS "pepys_Contacts_time", pepys."Con...
                                                             ^
HINT:  Perhaps you meant to reference the column "Contacts.bearing".


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/baris/.pyenv/versions/3.8.1/lib/python3.8/runpy.py", line 193, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/Users/baris/.pyenv/versions/3.8.1/lib/python3.8/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/Users/baris/Documents/PycharmProjects/pepys-import/pepys_import/import.py", line 49, in <module>
    main(path=args.path, archive=args.archive)
  File "/Users/baris/Documents/PycharmProjects/pepys-import/pepys_import/import.py", line 28, in main
    processor.process(path, data_store, True)
  File "/Users/baris/Documents/PycharmProjects/pepys-import/pepys_import/file/file_processor.py", line 156, in process
    contacts_sum = TableSummary(
  File "/Users/baris/Documents/PycharmProjects/pepys-import/pepys_import/core/store/table_summary.py", line 21, in __init__
    self.table_summary()
  File "/Users/baris/Documents/PycharmProjects/pepys-import/pepys_import/core/store/table_summary.py", line 24, in table_summary
    number_of_rows = self.session.query(self.table).count()
  File "/Users/baris/pepys-import/lib/python3.8/site-packages/sqlalchemy/orm/query.py", line 3653, in count
    return self.from_self(col).scalar()
  File "/Users/baris/pepys-import/lib/python3.8/site-packages/sqlalchemy/orm/query.py", line 3377, in scalar
    ret = self.one()
  File "/Users/baris/pepys-import/lib/python3.8/site-packages/sqlalchemy/orm/query.py", line 3347, in one
    ret = self.one_or_none()
  File "/Users/baris/pepys-import/lib/python3.8/site-packages/sqlalchemy/orm/query.py", line 3316, in one_or_none
    ret = list(self)
  File "/Users/baris/pepys-import/lib/python3.8/site-packages/sqlalchemy/orm/query.py", line 3389, in __iter__
    return self._execute_and_instances(context)
  File "/Users/baris/pepys-import/lib/python3.8/site-packages/sqlalchemy/orm/query.py", line 3414, in _execute_and_instances
    result = conn.execute(querycontext.statement, self._params)
  File "/Users/baris/pepys-import/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 982, in execute
    return meth(self, multiparams, params)
  File "/Users/baris/pepys-import/lib/python3.8/site-packages/sqlalchemy/sql/elements.py", line 293, in _execute_on_connection
    return connection._execute_clauseelement(self, multiparams, params)
  File "/Users/baris/pepys-import/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 1095, in _execute_clauseelement
    ret = self._execute_context(
  File "/Users/baris/pepys-import/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 1249, in _execute_context
    self._handle_dbapi_exception(
  File "/Users/baris/pepys-import/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 1476, in _handle_dbapi_exception
    util.raise_from_cause(sqlalchemy_exception, exc_info)
  File "/Users/baris/pepys-import/lib/python3.8/site-packages/sqlalchemy/util/compat.py", line 398, in raise_from_cause
    reraise(type(exception), exception, tb=exc_tb, cause=cause)
  File "/Users/baris/pepys-import/lib/python3.8/site-packages/sqlalchemy/util/compat.py", line 152, in reraise
    raise value.with_traceback(tb)
  File "/Users/baris/pepys-import/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 1245, in _execute_context
    self.dialect.do_execute(
  File "/Users/baris/pepys-import/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 588, in do_execute
    cursor.execute(statement, parameters)
sqlalchemy.exc.ProgrammingError: (psycopg2.errors.UndefinedColumn) column Contacts._bearing does not exist
LINE 2: ..., pepys."Contacts".time AS "pepys_Contacts_time", pepys."Con...
                                                             ^
HINT:  Perhaps you meant to reference the column "Contacts.bearing".

[SQL: SELECT count(*) AS count_1 
FROM (SELECT pepys."Contacts".contact_id AS "pepys_Contacts_contact_id", pepys."Contacts".name AS "pepys_Contacts_name", pepys."Contacts".sensor_id AS "pepys_Contacts_sensor_id", pepys."Contacts".time AS "pepys_Contacts_time", pepys."Contacts"._bearing AS "pepys_Contacts__bearing", pepys."Contacts"._rel_bearing AS "pepys_Contacts__rel_bearing", pepys."Contacts"._freq AS "pepys_Contacts__freq", pepys."Contacts"._range AS "pepys_Contacts__range", pepys."Contacts"._location AS "pepys_Contacts__location", pepys."Contacts"._elevation AS "pepys_Contacts__elevation", pepys."Contacts"._major AS "pepys_Contacts__major", pepys."Contacts"._minor AS "pepys_Contacts__minor", pepys."Contacts"._orientation AS "pepys_Contacts__orientation", pepys."Contacts".classification AS "pepys_Contacts_classification", pepys."Contacts".confidence AS "pepys_Contacts_confidence", pepys."Contacts".contact_type AS "pepys_Contacts_contact_type", pepys."Contacts"._mla AS "pepys_Contacts__mla", pepys."Contacts"._soa AS "pepys_Contacts__soa", pepys."Contacts".subject_id AS "pepys_Contacts_subject_id", pepys."Contacts".source_id AS "pepys_Contacts_source_id", pepys."Contacts".privacy_id AS "pepys_Contacts_privacy_id", pepys."Contacts".created_date AS "pepys_Contacts_created_date" 
FROM pepys."Contacts") AS anon_1]
(Background on this error at: http://sqlalche.me/e/f405)
```

**Comment by robintw** [comment]
I've just realised that I've missed out something rather crucial in the work I did to move to properties from attributes: with the code as it is currently the actual column names in the database have changed to things like `_bearing` rather than `bearing` - I forgot to add the bit in the SQLAlchemy class definitions to specify a specific name (`bearing`) even though the member variable is actually called something different (`_bearing`).

Sorry about this: I'll make a fix in a new PR ASAP.

**Comment by robintw** [comment]
Also, if you're running this against a Postgres database, as it looks like you are, then you'll need to create the Contact.range column manually (or just delete the db and recreate it through pepys-import) as this was added in my recent PRs.

**Comment by BarisSari** [comment]
Thanks @robintw for the answers. I actually tried it on both Postgres and SQLite. However, I see that you already opened a PR for a hotfix. I understand now why we have a problem. I'll recreate the Postgres DB.

**Comment by BarisSari** [comment]
Fixed in #243
